### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by nchervela**

### CODE REVIEW

**Comment by jeff-yin** [comment]
Document needs proper markdown formatting throughout

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Needs markdown throughout document

**Comment by jeffyin-dell** [comment]
@nchervela any update?

**Comment by nchervela** [comment]
Per email below, Amy added the markdowns.  thanks!
=============================
I’ve added the file in markdown 

Amy Buck
Technical Content Developer 3 | 2018 Dell Champion

**Comment by nchervela** [comment]
@jyin - Done.  thanks.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+This document describes SNMP agent implementation for
+OPX.  The engine used is PySNMP, which is
+a lightweight Python-based extensible agent with rich functionality.  The agent implementation is also in Python.
+Version 2 is currently supported, and will continue to be the default version
+of the agent.  Version 3 support will be
+available in the next release. The infrastructure supports implementation of
+any number of standard and proprietary mibs, and users can extend it to support
+traps, snmpsets etc.
+
+
```
Comment (jeffyin-dell) [inline README.md]: nitpick: Why so much whitespace?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+## Code location on switch
+
+
+
+ 
+
+
+
+compiled MIPS:                 /usr/lib/python2.7/dist-packages/opx-snmp
```
Comment (jeffyin-dell) [inline README.md]: First character of each line should be capitalized

File: README.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+## Implemented mibs:
+
+
+
+ 
+
+
+
+interface table
```
Comment (jeffyin-dell) [inline README.md]: This should be a bulleted list, and the first character should be capitalized

File: README.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
+# opx-snmp
+
+
+
+This document describes SNMP agent implementation for
```
Comment (jeffyin-dell) [inline README.md]: nitpick: don't think you need these manual line breaks

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]

**Comment by nchervela** [comment]
@jyin addressed review comments.

**Comment by jeffyin-dell** [comment]
@amybuck please review

**Review by jeffyin-dell — DISMISSED** [review]

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
I wrote in an e-mail that you're still missing a couple of the changes suggested by Amy. ALSO, the commit signoff is missing! Did you commit using `git commit -s` before opening the pull-request?
https://github.com/open-switch/opx-docs/wiki/Contributor-guide

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Still no sign-off :(

File: README.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
+# opx-snmp
+
+
+
+This file describes the SNMP agent implementation for OpenSwitch OPX.  The engine used is PySNMP, which is a lightweight Python-based extensible agent with rich functionality.  The agent implementation is also available in Python. Version 2 is currently supported, and will continue to be the default version of the agent.  Version 3 support will be available in the next release. The infrastructure supports implementation of any number of standard and proprietary MIBs, and users can extend it to support traps, snmpsets, and so on.
```
Comment (jeffyin-dell) [inline README.md:5]: Suggest changing "This file describes the SNMP agent implementation for OpenSwitch OPX." to "This is the SNMP agent for OpenSwitch OPX."

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]

**Comment by jeffyin-dell** [comment]
@nchervela please advise when you have a pull-request created with a signed commit. We will close this in favor of that one.

**Comment by jeffyin-dell** [comment]
@nchervela update?

**Pull request closed without merging**